tools/travis-scripts/run-clang-tidy.py

```diff
@@ -0,0 +1,329 @@
+#!/usr/bin/env python
```

Comment:
This script is delivered with clang-tidy. Please do not copy it to cocos2d-x source code.

Reply:
The script only shows warnings. It doesn't return a error to the shell. 

More specifically, `clang-tidy` has a option `-warnings-as-errors=<string>`, but I couldn't find it in the `run-clang-tidy.py ` script. Do you know any way to do it without modifying the script?

I just find other people doing the same, https://github.com/Sarcasm/irony-mode/blob/82ba45ec15c9011bbdf1d69cf25c8193d33c0028/server/build-aux/run-clang-tidy/run-clang-tidy.py

Reply:
Please create .clang-tidy file.
E.g. with "clang-tidy -dump-config".
In .clang-tidy file there is WarningAsErrors field. Please set it to '*'

Reply:
I see, thank you.